### DISCUSSION TITLE
ImmersionRC Ghost Tx Screen Support

### DISCUSSION THREAD

**Discussion by jstromsoe** [ExpressLRS/ExpressLRS#1484] [Ideas]
The screen on IRC Ghost Tx is useful for choosing options on startup and monitoring status.  I recall BrandonRC saying that this was under developed after watching his video on how to flash it, but alas I haven't seen a feature request for that or a bug on the subject.  Just an idea, anyone know if this is handled in another issue ID ?

**Comment by pkendall64** [comment]
Pretty sure this was all done in 2.2.0.

**Reply by jstromsoe** [reply]
I flashed the 'develop' branch thinking that had bleeding edge, didn't really stare at the individual commits.  After power up screen does nothing.  Does it have to have a Rx connection before anything happens ?  Is there any docs to read on it ?

**Reply by pkendall64** [reply]
'develop' is a very old branch. Main development is on master, but the OLED stuff was all merged for the release of 2.2.0 and I'm pretty sure Brandon tests on Ghost hardware.

**Reply by brandonrc** [reply]
Yea, you want to flash the latest official release.  I plan on adding some more OLED status items eventually but for now you have full control over power, rate, and telemetry. 

That youtube video is super out of date 🤣

**Reply by jstromsoe** [reply]
Aww man.  Might be worth a re-edit or something in the description that says use latest release.  Will reflash everything and see how it goes.

**Reply by jstromsoe** [reply]
Reflashed Tx and both my Rx's.  Now OLED works, but for some reason when I use the OLED menu to switch to high power, 250mW is the max (thought Ghost goes to 350mW) but also after power cycle it goes back to 50mW?  Weird.

EDIT: just read in the docs that after arming if no telemetry is sent that it will automagically adjust to the highest possible power...

Hopefully I'm right about that:

https://www.expresslrs.org/2.0/software/dynamic-transmit-power/
